packages/api/client.ts

```diff
@@ -407,32 +421,85 @@ export const getServiceStops = async (dbClient: Kysely<Database>, input: Service
         .where("toStop.stopType", "not in", ignoredStopTypes)
         .where((qb) => qb.where("fromStop.status", "=", "active").orWhere("toStop.status", "=", "active"))
         .$if(!!input.modes?.[0], (qb) => qb.where("services.mode", "in", input.modes ?? ["---"]))
-        .$if(!!input.busStopType, (qb) =>
-            qb
-                .where("toStop.busStopType", "=", input.busStopType ?? "---")
-                .where("fromStop.busStopType", "=", input.busStopType ?? "---"),
-        )
         .$if(!!input.stopTypes?.[0], (qb) =>
             qb
                 .where("fromStop.stopType", "in", input.stopTypes ?? ["---"])
-                .where("toStop.stopType", "in", input.stopTypes ?? ["---"]),
+                .where("toStop.stopType", "in", input.stopTypes ?? ["---"])
+                .$if(!!input.busStopType, (qb) =>
+                    qb
+                        .where("toStop.busStopType", "=", input.busStopType ?? "---")
```

Comment:
what if i just want to use the api to filter by busStopType then this will prevent me from doing so I would move this out of the if statement

Reply:
done